nats.go

```diff
@@ -701,14 +701,15 @@ func DrainTimeout(t time.Duration) Option {
 }
 
 // DisconnectErrHandler is an Option to set the disconnected error handler.
-func DisconnectedErrHandler(cb ConnErrHandler) Option {
+func DisconnectErrHandler(cb ConnErrHandler) Option {
 	return func(o *Options) error {
 		o.DisconnectedErrCB = cb
 		return nil
 	}
 }
 
 // DisconnectHandler is an Option to set the disconnected handler.
+// DEPRECATED: Use Use DisconnectErrHandler.
```

Comment:
Sorry to be a pain: `Use Use` one too many ;-)

Reply:
oh, my bad, cleaned